payments/vipps/seamless-view.md

```diff
@@ -18,114 +18,42 @@ sidebar:
 
 
 {% include jumbotron.html body="The **Seamless View** scenario gives your
-customers the opportunity to pay with Vipps directly within your webshop." %}
-
-## Introduction
-
-Vipps is a two-phase payment instrument supported by the major Norwegian banks.
+customers the opportunity to pay with Vipps directly within your webshop.
 In the Seamless View scenario, Swedbank Pay receives a mobile number (MSISDN)
 from the payer through Swedbank Pay Payments. Swedbank Pay performs a payment
-that the payer must confirm through the Vipps mobile app.
+that the payer must confirm through the Vipps mobile app." %}
 
 ![steps of the vipps purchase flow][vipps-purchase-flow]{:width="1200px" :height="500px"}
 
-## Purchase flow
-
-The sequence diagram below shows the two requests you have to send to
-Swedbank Pay to make a purchase.
-The links will take you directly to the API description for the specific
-request.
-
-```mermaid
-sequenceDiagram
-    Browser->>Merchant: start purchase (pay with VIPPS)
-    activate Merchant
-    Merchant->>-SwedbankPay: POST /psp/vipps/payments ①
-    activate SwedbankPay
-    note left of Merchant: First API request
-    SwedbankPay-->>-Merchant: rel: view-payment ②
-    activate Merchant
-    Merchant-->>-Browser: authorization page
-    activate Browser
-    note left of Browser: Open iframe ③
-    Browser->>Browser: Enter mobile number ④
-    Browser-->>-SwedbankPay: Passing data for authorization
-    activate SwedbankPay
-
-    SwedbankPay-->>-Vipps.API: POST <rel:create-auhtorization> ⑤
-    activate Vipps.API
-    Vipps.API-->>-SwedbankPay: response
-    activate SwedbankPay
-    SwedbankPay-->>-Browser: Authorization response (State=AwaitingActivity) ⑥
-    activate Browser
-    note left of Browser: check your phone
-
-    Vipps.API-->>Vipps_App: Confirm Payment UI
-    activate Vipps_App
-    note left of Vipps.API: Dialogue with Vipps ⑦
-    Vipps_App-->>Vipps_App: Confirmation Dialogue
-    Vipps_App-->>-Vipps.API: Confirmation
-    activate Vipps.API
-    Vipps.API-->>-SwedbankPay: make payment
-    activate SwedbankPay
-    SwedbankPay-->>SwedbankPay: execute payment
-    SwedbankPay-->>-Vipps.API: response
-    activate Vipps.API
-    Vipps.API-->>-SwedbankPay: authorize result
-    activate SwedbankPay
-    SwedbankPay-->>-Browser: Display authorize result
-```
+## Vipps seamless view integration flow
 
-### Explainations
-
-* ① When the payer starts the purchase process, you make a `POST` request
+1. When the payer starts the purchase process, you make a `POST` request
   towards Swedbank Pay with the collected Purchase information.
-* ② `rel: view-payment` is a value in one of the operations, sent as a response
+1. `rel: view-payment` is a value in one of the operations, sent as a response
```

Comment:
This list doesn't make much sense if it's not viewed in close proximity of the removed sequence diagram. The numbers in the list are directed directly towards the numbers in the diagram.